spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -75,13 +83,13 @@ public static void checkToRun() {
 	public void deleteAll() {
 		this.datastoreTemplate.deleteAll(EmbeddableTreeNode.class);
 		this.datastoreTemplate.deleteAll(AncestorEntity.class);
-		this.datastoreTemplate.deleteAll(AncestorEntity.DescendatEntry.class);
+		this.datastoreTemplate.deleteAll(AncestorEntity.DescendantEntry.class);
 		this.datastoreTemplate.deleteAll(TreeCollection.class);
 		this.testEntityRepository.deleteAll();
 	}
 
 	@Test
-	public void testSaveAndDeleteRepository() {
+	public void testSaveAndDeleteRepository() throws InterruptedException {
```

Comment:
This test is really a bunch of different, but related tests. Stylistically, my preference would be to break it up into multiple tests methods.

Reply:
Please make an issue for it. Thanks!

Reply:
https://github.com/spring-cloud/spring-cloud-gcp/issues/1154